lib/moray_client.js

```diff
@@ -49,7 +50,7 @@ function createClient(options, callback) {
                 url: pnodeUrl
             }, 'creating moray client');
 
-            var morayargs = clone(options.morayOptions);
+            var morayargs = jsprim.deepCopy(options.morayOptions);
```

Comment:
We already had jsprim as a dep. Changing this one usage allowed me to drop the 'clone' dep altogether.